pilot/pkg/networking/core/v1alpha3/cluster.go

```diff
@@ -605,6 +605,18 @@ func applyOutlierDetection(c *cluster.Cluster, outlier *networking.OutlierDetect
 		out.MaxEjectionPercent = &wrappers.UInt32Value{Value: uint32(outlier.MaxEjectionPercent)}
 	}
 
+	if outlier.SplitExternalLocalOriginErrors {
+		out.SplitExternalLocalOriginErrors = true
+		if outlier.ConsecutiveLocalOriginFailures.GetValue() > 0 {
+			out.ConsecutiveLocalOriginFailure = &wrappers.UInt32Value{Value: outlier.ConsecutiveLocalOriginFailures.Value}
+		}
+		// When split external origin errors is set to true, we should disable consecutive 5xx and gateways.
+		out.EnforcingConsecutive_5Xx = &wrappers.UInt32Value{Value: 0}
```

Comment:
is it not valid to have both set? I may want 5 errors for local origin and 10 for remote?

If not, I think we should add this to validation, why not just reject it?

Reply:
>  I may want 5 errors for local origin and 10 for remote?
Yes. We can do that. Actually above if we set to 0 we are ensuring that EnforcingConsecutive_5Xx is set to 0. So this is not needed. I will remove it.